src/index.js

```diff
@@ -46,47 +44,35 @@ module.exports = {
     'id-length': ['error', {
       exceptions: ['_', 'e', 'i']
     }],
+    'id-match': ['error', '^[_a-zA-Z][a-zA-Z0-9]*$|^[A-Z][A-Z0-9]*_[A-Z0-9]+$'],
```

Comment:
The `jscs` equivalent rule _"requireCamelCaseOrUpperCaseIdentifiers"_ allows both `_camelCase` and `camelCase_`, this regex allows only `_camelCase`.

If we are aiming for a port, I would allow `camelCase_` as well. 
If we are also improving I would remove support for both cases as we don't use any of them.

/cc @nunofgs @ruimarinho @rplopes